src/config/logger.ts

```diff
@@ -30,6 +31,13 @@ const requestSerializer = (req: Request) => (!req || !req.socket) ? req : {
 	body: req.body
 };
 
+const jobSerializer = (job: Job) => job && {
+	...job,
+	// Removing potentially extremely large amount of data from logs
+	data: undefined,
+	queue: undefined
```

Comment:
Why do we remove "queue" name from logs? 

It doesn't hold large amount of data, it is just a queue name
![image](https://user-images.githubusercontent.com/42425541/140675284-e5df7a8a-1c8f-4b49-b875-d1e69cc9b260.png)

Also instead of overriding this in serialiser we should properly remove this from where it is being logged.
However it is not a scope of this PR, so should be done in a separate one

- [ ] remove unrelated logging changes from this PR

Reply:
That's different: `logger.info({queue: queueName, queueMetrics: jobCounts}` we're still logging queue name, I'm just removing the queue object when serializing a whole job object because it also has redis information and the likes.  I could keep the queue name though.